tqdm/_tqdm.py

```diff
@@ -83,7 +83,7 @@ def print_status(self, s):
 
 
 def tqdm(iterable, desc=None, total=None, leave=False, file=sys.stderr,
-         ncols=None, mininterval=0.1, miniters=None):
+         ncols=None, mininterval=0.1, miniters=1, disable=False):
```

Comment:
this changes miniters default value - is it intentional? Docs for miniters are also changed here, but not in README